src/input.cpp

```diff
@@ -180,6 +197,8 @@ void input_manager::init()
     }
 }
 
+static constexpr int current_keybinding_verion = 1;
```

Comment:
```suggestion
static constexpr int current_keybinding_version = 1;
```
Or what is the meaning of the word "verion"?